lib/ansible/modules/apt.py

```diff
@@ -35,9 +35,9 @@
   update_cache:
     description:
       - Run the equivalent of C(apt-get update) before the operation. Can be run as part of the package installation or as a separate step.
-      - Default is not to update the cache.
     aliases: [ update-cache ]
     type: bool
+    default: 'no'
```

Comment:
this is incorrect, the actual default is `None` (which we normally omit). While it does function as if it where `no`/`false` there is a distinction as the module can make a difference between an 'explicit no' and 'implicit no'.

Reply:
Thanks for your explanation :)